tmk_core/common/keyboard.c

```diff
@@ -326,11 +330,8 @@ void keyboard_task(void) {
     housekeeping_task_kb();
     housekeeping_task_user();
 
-#if defined(OLED_DRIVER_ENABLE) && !defined(OLED_DISABLE_TIMEOUT)
     uint8_t ret = matrix_scan();
```

Comment:
‘ret’ is bad name. :-)
I like ‘matrix_changed’  or something good.

Reply:
I agree here.  A more meaningful name would be nice to have.

Reply:
`matrix_changed` it is.